spec/requests/api/v1.1/portfolios_controller_spec.rb

```diff
@@ -82,19 +82,21 @@
 
     context "with sort_by" do
       let!(:new_portfolio) { create(:portfolio, :owner => "a very new owner") }
+      let(:default_names_list) { [portfolio.name, new_portfolio.name].sort }
+      let(:owners_list) { [portfolio.owner, new_portfolio.owner].sort }
 
-      it "sorts when specified" do
+      it "sorts by owner when specified" do
         get "#{api_version}/portfolios?sort_by=owner", :headers => default_headers
 
         expect(json["meta"]["count"]).to eq 2
-        expect(json["data"].first["owner"]).to eq new_portfolio.owner
+        expect(json['data'].collect { |item| item['owner'] }). to match(owners_list)
```

Comment:
Overall I'm good with this change, just asking that you remove the space between "." and "to".